PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp

```diff
@@ -3052,14 +3075,11 @@ int Progress::thread()
 
 int Progress::createProgressWindow()
 {
-	DWORD styleEx = WS_EX_OVERLAPPEDWINDOW;
-	if (_hOwner)
-		styleEx |= WS_EX_TOOLWINDOW;
-
-	_hwnd = ::CreateWindowEx(styleEx,
+	_hwnd = ::CreateWindowEx(
+		WS_EX_TOOLWINDOW | WS_EX_OVERLAPPEDWINDOW | WS_EX_TOPMOST,
 		cClassName, _header, WS_POPUP | WS_CAPTION,
 		CW_USEDEFAULT, CW_USEDEFAULT, CW_USEDEFAULT, CW_USEDEFAULT,
-		_hOwner, NULL, _hInst, (LPVOID)this);

```

Comment:
> I was curious, because I know that the standard guarantees that any pointer (to an object) can implicitly convert to an (identically const/volatile) void*, no explicit (void*) needed.

I didn't know that, thanks.
Do you think that making explicit something that is implicit in this case changes something?

For example if we have

`void GetNumber(int num);`
`...`
`short int i = 8;`
`GetNumber(i);`
`GetNumber((int)i);`

changes nothing really.

Of course if we have

`void* i = &data;`
`GetNumber((int)i);`

that's a different thing.

I don't think the explicit cast here can do any harm anyway so I won't bother making new patch-set.